crowdsec-docs/docs/contributing/hub.md

```diff
@@ -1,40 +1,50 @@
 ---
 id: contributing_hub
 title: Hub
-sidebar_position: 2
+sidebar_position: 3
 ---
 
-# Contribute to the Hub
+Parsers, Scenarios, Collections allow Crowdsec to detect and block malevolent behavior.
 
-## Parsers and Scenarios Creation
+Supporting new services or improving the detection capabilities on existing software is a great way to contribute.
 
-Refer to the dedicated documentation to create [parsers](/docs/parsers/create/) and [scenarios](/docs/scenarios/create/).
-Before submitting your contribution, a few things should be checked :
+Sharing your parsers, scenarios and collections on the hub allows other users to use them to protect themselves.
 
-1. Collection(s)
+# Contributing
 
-It often makes sense for a new parser or scenario to be added to an existing [collection](/docs/next/collections/format), or create a new one.
+Anyone can open an issue about parsers/scenarios, or contribute a change with a pull request (PR) to the crowdsecuity/hub GitHub repository. You need to be comfortable with git and GitHub to work effectively.
 
-If your parsers and/or scenarios cover a new or specific service, having a dedicated collection for this service makes sense.
-In other cases, having a parser for `SpecificWebServer` access logs would justify a collection as it might as well include [all the default http related scenarios](https://hub.crowdsec.net/author/crowdsecurity/collections/base-http-scenarios).
+To get involved :
+
+- Have a look at open [issues](https://github.com/crowdsecurity/hub/issues) and [pull requests](https://github.com/crowdsecurity/hub/pulls)
+- Clone the hub repository
+- Create/Modify parsers/scenarios/collections
+- Create/Modify tests to ensure proper coverage
+- Open a pull request
 
+# Guidelines
 
-2. Tests
+## Technical Documentation
```

Comment:
you can also mention this blog article: https://crowdsec.net/blog/how-to-write-crowdsec-parsers-and-scenarios/ which i think is better to understand how to write a parser/scenario